core/commands/quick_stage.py

```diff
@@ -12,7 +12,8 @@
 CLEAN_WORKING_DIR = "Nothing to commit, working directory clean."
 ADD_ALL_UNSTAGED_FILES = " ?  All unstaged files"
 ADD_ALL_FILES = " +  All files"
-
+STAGED = "--- {} files are staged for commit. ---"
+COMMIT = "    git: commit"
 
```

Comment:
I use SublimeLinter and the [PEP8](https://github.com/SublimeLinter/SublimeLinter-pep8) plugin.  I think you can also just `pip3 install pep8` and then run `pep8 FILENAME` from the terminal.